dolphinscheduler-alert/src/main/java/org/apache/dolphinscheduler/alert/AlertServer.java

```diff
@@ -73,4 +76,37 @@ public static void main(String[] args){
         alertServer.start();
     }
 
+    /**
+     * send alert message
+     * @param alerts waiting list
+     */
+    private void send(List<Alert> alerts) {
```

Comment:
Suggestion:
AlerServer should not has this send, put it to other class

Reply:
yes, I want to remove the logic of regularly scanning the DB in the second step and then decouple the AlertServer class and the Send method.

Reply:
> yes, I want to remove the logic of regularly scanning the DB in the second step and then decouple the AlertServer class and the Send method.

Merging your pr will affect all the developer.  Even though refactor can split into more steps, but every step should code nice .

Reply:
hmm, I will abstract the Send function